### ISSUE TITLE
Multi Device Network Ports - 1 Week and 1 Month Graphs Broken

### ISSUE THREAD

**Issue by DuncCNI** [librenms/librenms#16378]
### The problem

Over the last several months, I want to say first confirmed observation was in late June, we've noticed that the 1 Week and 1 Month graphs located under any device with a "Ports" panel being populated are totally broken. Behavior has been replicated over dozens of daily builds over the last few months on at least half a dozen device types.

**Devices**:  APC, Tripplite, Eltek, Mellanox, Cisco, Linux (Ubuntu/CentOS/Alma), Dell iDRAC, and HPE iLO
**Browsers**: Windows 11 23H2 Firefox v130 & Chrome v128.0.6613.120, Ubuntu 22.04 LTS Firefox Snap v129.0.2
**Approx Start Time**: May-June 2024, not super confident in running version before issue began.

Below are my observations thus far: 

The issue can be observed currently on any device with _network interfaces_ while under the corresponding device's `/device/$id/ports/graphs?type=$foo` pages. After some experimenting in the GUI Debug panel, I'm able to report the pictured `1d/1w/1m/1y` string of graphs from the pictured server are generated with the below URLs for Device "Ports" and "Health".

```
#Ports - dwmy
/graphs/from=-1d/id=6163724/type=port_bits  #working
/graphs/from=-1w/id=6163724/type=port_bits  #broken
/graphs/from=-1m/id=6163724/type=port_bits  #broken
/graphs/from=-1y/id=6163724/type=port_bits  #working
#Health - epoch
/graphs/type=device_processor/device=711/from=1725827400/ 
/graphs/type=device_processor/device=711/from=1725309000/
/graphs/type=device_processor/device=711/from=1723235400/
/graphs/type=device_processor/device=711/from=1694377800/
```

Testing the following edits in-browser to the URLs seems to correct them without issue:
```
#Week
/graphs/from=-1week/id=6163724/type=port_bits
/graphs/from=-1weeks/id=6163724/type=port_bits
/graphs/from=-7d/id=6163724/type=port_bits
#Month
/graphs/from=-1month/id=6163724/type=port_bits
/graphs/from=-1months/id=6163724/type=port_bits
/graphs/from=-30d/id=6163724/type=port_bits
```

>Live Behavior (Mellanox Pictured):

![image](https://github.com/user-attachments/assets/6cdebe0e-7cbe-4246-bdac-b545defe0ad8)
![image](https://github.com/user-attachments/assets/68771416-8df2-4038-9c65-4adab569efd2)

>Edit of broken fields corrected within F12 scope

![image](https://github.com/user-attachments/assets/44d03e0f-b1cc-4c86-abc4-9bedb4da6354)

TL;DR - as of ~Q2 2024 -> `1w == ~3 Hours` and `1m == ~1 minute`?

### Output of ./validate.php

```txt
===========================================
Component | Version
--------- | -------
LibreNMS  | 24.8.0-57-g30b174508 (2024-09-08T18:00:42-04:00)
DB Schema | 2024_08_27_182000_ports_statistics_table_rev_length (299)
PHP       | 8.3.8
Python    | 3.10.12
Database  | MariaDB 10.6.18-MariaDB-0ubuntu0.22.04.1
RRDTool   | 1.7.2
SNMP      | 5.9.1
===========================================

[OK]    Composer Version: 2.7.9
[OK]    Dependencies up-to-date.
[OK]    Database connection successful
[OK]    Database connection successful
[OK]    Database Schema is current
[OK]    SQL Server meets minimum requirements
[OK]    lower_case_table_names is enabled
[OK]    MySQL engine is optimal
[OK]    Database and column collations are correct
[OK]    Database schema correct
[OK]    MySQL and PHP time match
[OK]    Active pollers found
[OK]    Dispatcher Service not detected
[OK]    Locks are functional
[OK]    Python poller wrapper is polling
[OK]    Redis is unavailable
[OK]    rrdtool version ok
[OK]    Connected to rrdcached
```

### Anything in the logs that might be useful for us?

Unfortunately, no, not for this particular issue.

**Comment by DuncCNI** [comment]
Confirmed this is still occurring as of `24.8.0-77-g375903025 - Thu Sep 19 2024 20:22:24 GMT-0400`

**Comment by laf** [comment]
Give #16416 a try, fixed it for me.

**Comment by DuncCNI** [comment]
> Give #16416 a try, fixed it for me.

Can confirm this resolves it!
